compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -1749,6 +1749,10 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
           dotc.core.Types.decorateTypeApplications(self).appliedTo(targs)
         def substituteTypes(from: List[Symbol], to: List[TypeRepr]): TypeRepr =
           self.subst(from, to)
+
+        def typeParams: List[TypeRepr] = self match
```

Comment:
This is getting the arguments and not the parameters, I would call it `typeArgs`.

Reply:
`typeParams` provides references to the definitions of the parameters. Here we provide the type arguments passed to the type constructor.

Reply:
Not convinced by the naming difference, but not strong opinion.